build.gradle

```diff
@@ -112,6 +116,8 @@ ext.libraries = [
         stethoOkHttp            : "com.facebook.stetho:stetho-okhttp:$versions.stetho",
         leakCanary              : "com.squareup.leakcanary:leakcanary-android:$versions.leakCanary",
         tinyDancer              : "com.github.brianPlummer:tinydancer:$versions.tinyDancer",
+        paperworkPlugin         : "hu.supercluster:paperwork-plugin:$versions.paperwork",
```

Comment:
Not if the library version definitions are extracted as discussed above. It would have the added benefit of not having to maintain version numbers for the plugin and the runtime in two different places.